syntax = "proto3";
package dpi;

message HelloMessage {
    string name = 1;
}
